crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -148,11 +158,21 @@ impl CallContractCommand {
 
 	/// If the contract has not been built, build it in release mode.
 	async fn ensure_contract_built(&self, cli: &mut impl Cli) -> Result<()> {
+		let path_flag = self.path.clone();
```

Comment:
This code is now being used in four different place, it makes sense to refactor this and create a method within the `pop-cli/common` module. Also will make it easier to test only this funcionality.